resources/views/tasks/editMobile.blade.php

```diff
@@ -0,0 +1,303 @@
+  </script>
+  <script src="{{mix('js/tasks/show.js')}}"></script>
+  <script>
+      const store = new Vuex.Store();
+      const main = new Vue({
+        mixins:addons,
+        store: store,
+        el: "#taskMobile",
+        data: {
+          //Edit data
+          fieldsToUpdate: [],
+          jsonData: "",
+          monacoLargeOptions: {
+            automaticLayout: true,
+          },
+          showJSONEditor: false,
+
+          // Reassignment
+          selected: null,
+          selectedIndex: -1,
+          usersList: [],
+          filter: "",
+          showReassignment: false,
+          task,
+          userHasAccessToTask,
+          statusCard: "card-header text-capitalize text-white bg-success",
+          selectedUser: [],
+          hasErrors: false,
+          redirectInProcess: false,
+          formData: {},
+          submitting: false,
+          userIsAdmin,
+          userIsProcessManager,
+          showTree: false,
+        },
+        watch: {
+          task: {
+            deep: true,
+            handler(task, oldTask) {
+              window.ProcessMaker.breadcrumbs.taskTitle = task.element_name;
+              if (task && oldTask && task.id !== oldTask.id) {
+                history.replaceState(null, null, `/tasks/${task.id}/edit`);
+              }
+            }
+          },
+        },
+        computed: {
+          taskDefinitionConfig () {
+            let config = {};
+            if (this.task.definition && this.task.definition.config) {
+              return JSON.parse(this.task.definition.config);
+            }
+            return {};
+          },
+          taskHasComments() {
+            const commentsPackage = 'comment-editor' in Vue.options.components;
+            let config = {};
+            if (commentsPackage && this.task.definition && this.task.definition.config) {
+              config = JSON.parse(this.task.definition.config);
+            }
+            return config;
+          },
+          dueLabel() {
+            const dueLabels = {
+              'open': 'Due',
+              'completed': 'Completed',
+              'overdue': 'Due',
+            };
+            return dueLabels[this.task.advanceStatus] || '';
+          },
+          isSelfService() {
+            return this.task.process_request.status === 'ACTIVE' && this.task.is_self_service;
+          },
+          dateDueAt () {
+            return this.task.due_at;
+          },
+          createdAt () {
+            return this.task.created_at;
+          },
+          completedAt () {
+            return this.task.completed_at;
+          },
+          showDueAtDates () {
+            return this.task.status !== "CLOSED";
+          },
+          disabled () {
+            return this.selectedUser ? this.selectedUser.length === 0 : true;
+          },
+          styleDataMonaco () {
+            let height = window.innerHeight * 0.55;
+            return "height: " + height + "px; border:1px solid gray;";
```

Comment:
If it is possible consider to use a bootstrap class like "boerder" instead of the hardcoded style `border:1px solid gray;`

Reply:
this is legacy code